amical/tests/test_extraction.py

```diff
@@ -22,21 +45,76 @@ def test_add_infos_simulated():
     assert infos["date-obs"] == hdr["DATE-OBS"]
     assert infos["telescop"] == hdr["TELESCOP"]
     assert "observer" not in infos  # Keys that are not in hdr should not be in infos
+    assert "observer" not in infos.hdr  # Keys that are not in hdr should still not be
 
 
-def test_add_infos_header_commentary():
+@pytest.mark.filterwarnings("ignore: Commentary cards")
```

Comment:
Let's make this as specific as possible
```suggestion
@pytest.mark.filterwarnings("ignore: Commentary cards are removed from the header")
```

Reply:
Do you want me to use the full warning message or is this new version enough?

Reply:
I merged the change, but I'll keep it "unresolved" just in case I need to add the full message.

Reply:
well I don't think we can/should use the full warning because it includes data determined at runtime, which is what I meant by "as specific as possible"